doc/source/releases/20.09_announce_user.rst

```diff
@@ -0,0 +1,184 @@
+
+===========================================================
+September 2020 Galaxy Release (v 20.09)
+===========================================================
+
+.. include:: _header.rst
+
+Highlights
+===========================================================
+
+Pluggable Upload Components - Dropbox & WebDav
+----------------------------------------------
+
+`@jmchilton <https://github.com/jmchilton>`__ has written a new pluggable interface for upload sources. He has initially added two new backends, you can now import data from Dropbox and any WebDav servers you might have access to. This work also makes it easier for developers to add support for other, new personal data sources in the future.
+
+An example of a Dropbox account:
+
+.. figure:: images/20.09-upload-dropbox.png
+   :alt: Screenshot of Dropbox with a "bams" folder and several test files
+
+There will be a new button for "Choose Remote files", when sources are configured, and then you will be able to see and search across all of the remote data locations to which you might have access.
+
+.. figure:: images/20.09-upload-search.png
+   :alt: Search across sources. A modal dialog showing a new "Dropbox" folder amongst "FTP Directory" and library location.
+
+And then you will be able to see your load your files easily in Galaxy!
```

Comment:
```suggestion
You will be able to see and load your files easily in Galaxy!
```
(why -"And then": if you read this right after the previous paragraph, you get two consecutive "and then you will be able to see and...")

Reply:
Just me completely forgetting what I wrote :sweat_smile: cheers